css-color/Overview.bs

```diff
@@ -1714,7 +1717,7 @@ Profiled, Device-dependent Colors</h2>
 	or an RGB colorspace (such as ProPhoto <!-- ref -->, <!-- would be good to mention AdobeRGB 1998 here too, need to check trademark and copyright issues first --> widely used by photographers), or any other color or monochrome output device which has been characterized.
 	In addition, for convenience,
 	CSS provides two predefined RGB color spaces:
-	DCI P3 [[!DCI-P3]],
+	p3d65 [[!DCI-P3]],
```

Comment:
I don’t know where the `P3` part comes from, but a more useful keyword for authors would probably be `cinema` (or `movie-theater`, `film-projector` …), alongside `uhdtv`, `hdtv`, `sdtv`, `ntsc`, `pal`/`ebu`, … ([table at Wikipedia](https://en.wikipedia.org/wiki/RGB_color_space#Specifications))